themes/cclw/constants/faqs.tsx

```diff
@@ -502,7 +502,7 @@ export const FAQS: TFAQ[] = [
         the NewClimate Institute and its partners for the{" "}
         <ExternalLink url="https://climatepolicydatabase.org/">Climate Policy Database</ExternalLink> project. Prior to their ingest into the
         database, documents were reviewed by researchers at Climate Policy Radar to ensure that no duplicate entries were added and that entries fell
-        within the scope of the Climate Change Laws of the World  <ExternalLink url="https://climatepolicydatabase.org/">methodology</ExternalLink>.
+        within the scope of the Climate Change Laws of the World <ExternalLink url="https://climate-laws.org/methodology">methodology</ExternalLink>.
```

Comment:
please use the `<LinkWithQuery>` component and just use the local url of `/methodology` because we are linking to an internal page